app/views/users/edit.html.erb

```diff
@@ -8,9 +8,18 @@
         </p>
 
         <%= form_for :user, url: { action: 'update' }, method: :patch do |f| %>
-          <%= render partial: 'users/dropdowns', locals: { f: f, u: @current_user } %>
+          <div id="form-first">
```

Comment:
I think it's best we revert these changes in this PR -- let's keep this PR focused on the timeline updates, I'll be fixing this up in #656